solr/solrj/src/java/org/apache/solr/common/cloud/ZkStateReader.java

```diff
@@ -103,6 +103,7 @@
   public static final String STATE_TIMESTAMP_PROP = "stateTimestamp";
   public static final String COLLECTIONS_ZKNODE = "/collections";
   public static final String LIVE_NODES_ZKNODE = "/live_nodes";
+  public static final String NODE_ROLES = "/node_roles";
```

Comment:
A comment here should explain the difference between `/node_roles` and `/roles.json` defined further down.

Reply:
Fixed, thanks.